internal/executor/instance/persistentworker/isolation/tart/tart.go

```diff
@@ -192,22 +188,47 @@ func (tart *Tart) Run(ctx context.Context, config *runconfig.RunConfig) (err err
 		break
 	}
 
+	tart.vm = vm
+
 	tart.logger.Debugf("IP %s retrieved from VM %s, running agent...", ip, vm.Ident())
```

Comment:
I think this should be in `Run()`, we're not running the agent here.

Reply:
Moved in [04fee80](https://github.com/cirruslabs/cirrus-cli/pull/698/commits/04fee80eef05acaff472362efe61466cd1985116)